### PR TITLE
Fixed a few minor issues and added clang ignore for mixpanelViewId

### PR DESCRIPTION
**Pull request by samgreen**

### CODE REVIEW

File: Mixpanel/MPNotificationViewController.m [hunk 1]
```diff
@@ -147,7 +147,7 @@ - (UIStatusBarAnimation)preferredStatusBarUpdateAnimation
     return UIStatusBarAnimationFade;
 }
 
-- (UIInterfaceOrientationMask)supportedInterfaceOrientations
+- (NSUInteger)supportedInterfaceOrientations
```
Comment (alex-hofsteede) [inline Mixpanel/MPNotificationViewController.m]: This was a warning?

File: Mixpanel/MPUIControlBinding.h [hunk 2]
```diff
@@ -16,8 +16,8 @@
 
 - (instancetype)init __unavailable;
 - (instancetype)initWithEventName:(NSString *)eventName
-                 onPath:(NSString *)path
-       withControlEvent:(UIControlEvents)controlEvent
-         andVerifyEvent:(UIControlEvents)verifyEvent NS_DESIGNATED_INITIALIZER;
+                           onPath:(NSString *)path
```
Comment (alex-hofsteede) [inline Mixpanel/MPUIControlBinding.h]: breaks indenting

File: Mixpanel/MPNotificationViewController.m (hunk 1 above)
Comment (samgreen) [inline Mixpanel/MPNotificationViewController.m]: - WARN  | [Mixpanel/Mixpanel]  /Users/samgreen/code/mixpanel-iphone/Mixpanel/MPNotificationViewController.m:150:1: warning: conflicting return type in implementation of 'supportedInterfaceOrientations': 'NSUInteger' (aka 'unsigned long') vs 'UIInterfaceOrientationMask' (aka 'enum UIInterfaceOrientationMask') [-Wmismatched-return-types]

Comment (samgreen) [inline Mixpanel/MPNotificationViewController.m]: Surprising huh?

File: Mixpanel/MPUIControlBinding.h (hunk 2 above)
Comment (alex-hofsteede) [inline Mixpanel/MPUIControlBinding.h]: oh wait never mind, I was lining up the deleted lines with the added lines

**Comment by alex-hofsteede** [comment]
ok LGTM

File: Mixpanel/MPUIControlBinding.h (hunk 2 above)
Comment (samgreen) [inline Mixpanel/MPUIControlBinding.h]: Yeah I fixed the indenting that was here

**Pull request merged**